src/pages/Beta/Bridge/TabulationBox/index.tsx

```diff
@@ -1,53 +1,41 @@
 import React, { useState } from 'react'
 import { QuestionBox, TableContent, FullBox } from '../styleds'
-import { TrollQuestionBox, TrollFeeBox, BridgingBox, OtherQuestionBox, GeneralBox } from '../QuestionBoxes'
 import { Text } from '@pangolindex/components'
+import { GeneralBox } from './QuestionBoxGeneral'
+import { Questions, useGetQuestions } from 'src/state/airdrop/hooks'
 
-export const QuestionAnswer = () => {
-  const [general, setGeneral] = useState<boolean>(true)
-  const [otherQuestion, setOtherQuestion] = useState<boolean>(false)
-  const [bridging, setBridging] = useState<boolean>(false)
-  const [trollQuestions, setTrollQuestions] = useState<boolean>(false)
-  const [trollFee, setTrollFee] = useState<boolean>(false)
-
-  const showGeneral = () => {
-    setGeneral(true)
-    setOtherQuestion(false)
-    setBridging(false)
-    setTrollQuestions(false)
-    setTrollFee(false)
-  }
-
-  const showOtherQuestion = () => {
-    setGeneral(false)
-    setOtherQuestion(true)
-    setBridging(false)
-    setTrollQuestions(false)
-    setTrollFee(false)
-  }
-
-  const showBridging = () => {
-    setGeneral(false)
-    setOtherQuestion(false)
-    setBridging(true)
-    setTrollQuestions(false)
-    setTrollFee(false)
-  }
+interface ShortArray {
+  id: number
+  subcategory: string
+}
 
-  const showTrollQuestions = () => {
-    setGeneral(false)
-    setOtherQuestion(false)
-    setBridging(false)
-    setTrollQuestions(true)
-    setTrollFee(false)
+export const QuestionAnswer = () => {
+  const [subcategory, setSubCategory] = useState<string>('General')
+  const [active, setActive] = useState<number>(0)
+  const { data: questions } = useGetQuestions('Bridge')
```

Comment:
instead of passing category in plain string like `Bridge`, `Airdrop` please create proper category enum and use it